<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>271</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,4,2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>第五远节指骨骺</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>期</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,4,2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>第五中节指骨骺</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>期</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,4,2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>第五近节指骨骺</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3"/>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>期</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,4,2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>第五掌骨骺</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_4"/>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>期</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,4,2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>第三远节指骨骺</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_5"/>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>期</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="5,4,2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>第三中节指骨骺</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_6"/>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>期</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="5,4,2">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>第三近节指骨骺</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_7"/>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>期</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>标准图谱阅读</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>标准图谱阅读</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>标准图谱阅读</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>标准图谱阅读</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>标准图谱阅读</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>标准图谱阅读</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>标准图谱阅读</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>527</width>
     <height>689</height>
    </rect>
   </property>
   <property name="title">
    <string>标准图谱调阅</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="3,1,3,1,3,1,3,1" rowminimumheight="3,1,3,1,3,1,3,1">
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(200,101,102);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_16">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(200,101,102);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="atla_1">
        <property name="text">
         <string>1期</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="atla1_2">
        <property name="text">
         <string>2期</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_17">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(200,101,102);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_18">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(200,101,102);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="atla1_3">
        <property name="text">
         <string>3期</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="atla1_4">
        <property name="text">
         <string>4期</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_19">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(200,101,102);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_20">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(200,101,102);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="atla1_5">
        <property name="text">
         <string>5期</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="atla1_6">
        <property name="text">
         <string>6期</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_21">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(200,101,102);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="atla1_7">
        <property name="text">
         <string>7期</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
